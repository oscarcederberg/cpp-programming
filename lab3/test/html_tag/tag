this <tag> should be removed</tag>
this <tag that spans
over two lines> should be removed as well