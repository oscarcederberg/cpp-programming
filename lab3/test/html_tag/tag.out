this  should be removed
this  should be removed as well